src/test/kotlin/com/igorwojda/integer/pyramidgenerator/challenge.kt

```diff
@@ -4,7 +4,14 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 fun generatePyramid(n: Int): List<String> {
-    TODO("not implemented")
+
+    val pyramidElementsList = mutableListOf<String>()
+    var lastRow = if (n == 2) n / 2 else (n - 1)
+    for (element in 0 until n) {
+        pyramidElementsList.add(" ".repeat(lastRow) + "#".repeat(element * 2 + 1) + " ".repeat(lastRow--))
```

Comment:
For readibility please split this statement into multiple lines. Also, do the post decrement as a separate statement. If you're not careful, it can have ugly side-effects when refactoring the code.